src/application.coffee

```diff
@@ -4,7 +4,7 @@ application
 Main application that create a Mn.Application singleton and exposes it. Needs
 router and app_layout view.
 ###
-_        = require 'underscore'
+_ = require 'underscore'
```

Comment:
I think this line is irrelevant for the scope of this PR, we should try to avoid edit/fix/refacto which are not correlated to the PR subject. You may revert, amend your commit and force push to update the branch and this PR, thanks. :)